src/web/lib/hakuneko/frontend@classic-dark/app.html

```diff
@@ -110,7 +123,10 @@
             </div>
             <hakuneko-jobs></hakuneko-jobs>
         </div>
-        <hakuneko-pages style$="[[ getPagePanelStyle(readerEnabled) ]]" selected-chapter="[[ chapter ]]" selected-media="{{ selectedMediaIndex }}"></hakuneko-pages>
+        <div class="content">
+            <hakuneko-start style$="[[ getStartPanelStyle(readerEnabled,chapter) ]]"></hakuneko-start>
```

Comment:
An alternative approach could be to use a frame and show the start page from e.g. `hakuneko.download/embed.html` or something (may also have different themed versions).
Disadvantage is probably seamless integration...

Reply:
It would hurt the eyes with the light theme as the site is dark themed